aspects/aspects.go

```diff
@@ -228,6 +237,89 @@ type accessPattern struct {
 	access accessType
 }
 
+// getPath returns a path if the specified name matches the access pattern's name
+// (with optional placeholders) and an empty string, if not.
+func (p *accessPattern) getPath(name string) (string, error) {
+	if strings.ContainsAny(p.name, "{}") {
+		matches := matchedPlaceholders(p.name, name)
+		if matches == nil {
+			// aspect name w/ placeholders doesn't match the specified name
+			return "", nil
+		}
+
+		return fillInPlaceholders(p.path, matches)
+	} else if p.name == name {
+		return p.path, nil
+	}
+
+	return "", nil
+}
+
+// matchedPlaceholders takes a dot-separated pattern with optional placeholders
+// (e.g., "foo.{bar}") and a name. If the name fulfils the pattern, it returns
```

Comment:
This part is a bit obscure to me, as I don't know what "fulfils the pattern" means. I don't know how this pattern works, it's obviously not a regexp but without some help I'm just unable to understand how it works.